src/oak_d.py

```diff
@@ -480,6 +483,15 @@ async def get_point_cloud(
         if not cls.worker.running:
             raise ViamError("get_point_cloud called before camera worker is ready.")
 
+        # By default, we do not get point clouds even when color and depth are both requested
+        # We have to reinitialize the worker/OakCamera to start making point clouds
+        if cls.worker.pcd is None:
```

Comment:
I'm not super clear on your question, but this block should only be hit the first time get_point_cloud is called, because otherwise `cls.worker.pcd` is not `None`